e2e/test.bats

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bats
+
+set -Eeuo pipefail
+
+# This tests that we get the same verification zip path when we run the commands:
+#
+# - config
+# - inspect -build-config -verification
+# - build -verification
+@test "verification output paths match" {
+	rm -rf dist/ out/
+	CONFIG="$(actions-go-build config | grep ZIP_PATH_VERIFICATION | grep -Eo '/.*$')"
+	BUILDENV="$(actions-go-build inspect --build-config --verification | jq -r .Paths.ZipPath)"
+	actions-go-build build -q
+	BUILD="$(actions-go-build build -q -verification -json | jq -r .Config.Paths.ZipPath)"
+
+	echo "$CONFIG"
+	echo "$BUILDENV"
+	echo "$BUILD"
+
+	diff <(echo "$CONFIG") <(echo "$BUILDENV")
+	diff <(echo "$CONFIG") <(echo "$BUILD")
```

Comment:
Nit: I think it would be easier to read this output with headings before each of the diffs.  A simple echo could work, but it might be a little awkward when the diff is empty.  Maybe something like the below?
```suggestion
	if [ "$CONFIG" != "$BUILDENV" ]; then
		echo "config vs. buildenv:"
		diff <(echo "$CONFIG") <(echo "$BUILDENV")
	fi
	if [ "$CONFIG" != "$BUILD" ]; then
		echo "config vs. build:"
		diff <(echo "$CONFIG") <(echo "$BUILD")
	fi
```